src/one_wire.toit

```diff
@@ -343,6 +344,15 @@ class Bus:
     crc.add bytes
     return crc.get-as-int
 
+default-rmt-channels_:
```

Comment:
Maybe change this to `default-rmt-channel-count_` or just add a return type so it is clear that this the number, not some structured list of objects you get back:

```suggestion
default-rmt-channels_ -> int:
```

Reply:
done.